### DISCUSSION TITLE
Ability to set a weekly schedule with specific days when Light and Dark Auto Switches.

### DISCUSSION THREAD

**Discussion by 16BitRun** [AutoDarkMode/Windows-Auto-Night-Mode#867] [Q&A] [answered]
Hi,

Is it possible to set a weekly schedule of when light mode with be on and when Dark Mode based on specific days on the week. For example, I work from home and I would like to automatically set Light Mode on my weekly Microsoft Teams Video Conferences on Mondays, Wednesdays and Fridays only from 7:30am - 2:30pm and then have it revert to Dark Mode. I know this is possible using Windows Task Manager but it would be great if this could be implemented into the Auto  Dark Mode app.

**Comment by Spiritreader** [comment, marked as answer]
We once had the idea to make switching more granular, but there's not enough demand for a feature like this when we consider the time we have available to develop something like this.
